### ISSUE TITLE
Importing localsccurs

### ISSUE THREAD

**Issue by ma7ma7** [g45t345rt/minifaker#9]
Hello,

When we try importing locals an error
```javascript
import 'minifaker/locales/en';
```
An error occurs in this example I used parcel
```
@parcel/core: Failed to resolve 'minifaker/locales/en' from './src/index.js'
```

My solution was importing the locals directaly:
```javascript
import '../node_modules/minifaker/dist/esm/locales/en';
```
I think is better to define a default local or add a method to set the locals

```javascript
minifaker.setLocal('en');
```

Thank you man 🙏

**Comment by ma7ma7** [comment]
I found in the type a `setDefaultLocale()` but doesn't work, is better to update the documentation please.

**Comment by g45t345rt** [comment]
hmmm I don't think parcel support conditionnal exports 
https://nodejs.org/api/packages.html#conditional-exports
I could be wrong though

In the mean time you can use
```ts
import 'minifaker/dist/esm/locales/en'
```

You don't have to use `setDefaultLocale()`

**Comment by ma7ma7** [comment]
Yeah, the import as you mention is working fine, I wish that if you found some time to update the documentation is gonna be great.

Thank you a lot man

**Comment by earthpyy** [comment]
I also found the similar problem when using `minifaker` in `jest` tests.

<img width="450" alt="CleanShot 2565-01-16 at 00 25 24@2x" src="https://user-images.githubusercontent.com/5132475/149631471-bc210007-621a-4bbb-a5d4-34fa94932536.png">

Solved by adding this line in `jest.config.js`

```js
transformIgnorePatterns: ['node_modules/(?!minifaker)']
```

And this line into `compilerOptions` in `tsconfig.spec.json`

```json
"allowJs": true
```

Not sure if it was related to this issue or not?

**Comment by Vanderscycle** [comment]
I also have this problem when running e2e tests on NestJs and that didn't fix it for me.
